src/components/BooksList.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function BooksList() {
+  return (
+    <div>BooksList</div>
+  );
+}
+
```

Comment:
- [ ] Then from here, You will need to have another `Books` component that renders the `BookList` and `NewBook` as children and it will be the one rendered in the `App`